### PR TITLE
Allow PDF to be find

### PR DESCRIPTION
**Pull request by Seb33300**
This make possible to work with PDF files when using `imagick` driver.

### CODE REVIEW

File: Imagine/Data/DataManager.php [hunk 1]
```diff
@@ -137,7 +137,7 @@ public function find($filter, $path)
             throw new \LogicException(sprintf('The mime type of image %s was not guessed.', $path));
         }
 
-        if (0 !== mb_strpos($binary->getMimeType(), 'image/')) {
+        if (0 !== mb_strpos($binary->getMimeType(), 'image/') && $binary->getMimeType() !== 'application/pdf') {
             throw new \LogicException(sprintf('The mime type of image %s must be image/xxx got %s.', $path, $binary->getMimeType()));
```
Comment (phorzycki) [inline Imagine/Data/DataManager.php]: The error message should be modified to reflect your change:

```
The mime type of image %s must be image/xxx or application/pdf, got %s.
```

Comment (Seb33300) [reply Imagine/Data/DataManager.php]: Fixed

**Comment by maximgubar** [comment]
Hi @Seb33300 ,
Thank you for the merge request,
could you please merge latest master in your branch in order to pass the CI ?

**Comment by fyrye** [comment]
If I am not mistaken, this will result in a file with the `image/png` mime-type  and `.pdf` extension. Until #686 is merged to resolve extension changes #584

**Comment by dbu** [comment]
can you please add a functional test in https://github.com/liip/LiipImagineBundle/blob/master/Tests/Functional/Controller/ImagineControllerTest.php so that we can be sure the response mime type corresponds to the output?

**Comment by Seb33300** [comment]
I rebased my pull request.
If someone has time to work on the functional test please do it.
I am not sure when I will be able to do it.

**Pull request closed without merging**